root/user/profile.tt

```diff
@@ -91,11 +91,15 @@
             [% END %]
         [% END %]
 
-        [% IF c.user_exists AND (viewing_own_profile OR c.user.is_account_admin);
-             WRAPPER property name=l('Last login:');
-               UserDate.format(user.last_login_date);
-             END;
-           END %]
+        [% IF c.user_exists AND (viewing_own_profile OR c.user.is_account_admin) %]
+           [% WRAPPER property name=l('Last login:') %]
```

Comment:
Because that is how all the other IF-ELSEs are done in this file, so it seemed more consistent.